File: app/src/main/java/de/sneak/sneakpeek/service/PredictionService.java
package de.sneak.sneakpeek.service;

import okhttp3.ResponseBody;
import retrofit2.http.GET;
import rx.Observable;

public interface PredictionService {

    String BASE_URL = "http://lebkuchenbande.de/";

    @GET("progno.txt")
    Observable<ResponseBody> getPredictions();

    @GET("studios.txt")
    Observable<ResponseBody> getStudios();

    @GET("actual.txt")
    Observable<ResponseBody> getPreviousMovies();
}

File: app/src/main/java/de/sneakpeek/service/PredictionService.kt
package de.sneakpeek.service

import io.reactivex.Observable
import okhttp3.ResponseBody
import retrofit2.http.GET

interface PredictionService {

    @get:GET("progno.txt")
    val moviePredictions: Observable<ResponseBody>

    @get:GET("studios.txt")
    val studios: Observable<ResponseBody>

    @get:GET("actual.txt")
    val actualMovies: Observable<ResponseBody>

    companion object {
        val BASE_URL = "http://lebkuchenbande.de/"
    }
}
